cms/static/coffee/spec/models/course_spec.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
I'm tempted to remove these comments if you can mostly address the issues recommended. I'd rather that the top comment describe the purpose of the test, rather than how the file was transpiled.